### ISSUE TITLE
Allow users to set github url

### ISSUE THREAD

**Issue by nholthaus** [HookyQR/VSCodeGitHubIssues#12]
Please add a configuration option for the github url to use. This would allow github enterprise users to configure their instance instead of defaulting to github.com

**Comment by obambrough** [comment]
I agree about supporting GitHub Enterprise. Is it possible to pull the URL from the project's .git/config file?

**Comment by zsirotto** [comment]
Any status on this?
